### DISCUSSION TITLE
2021-04-16 向邪恶的 SEO 低头，我部署了 static site 版本 ~ 蒋继发 (ThaddeusJiang)

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#27] [Announcements]
# 2021-04-16 向邪恶的 SEO 低头，我部署了 static site 版本 ~ 蒋继发 (ThaddeusJiang)

2021 年 04 月 16 日，我向邪恶的 SEO 低头了，😂

https://thaddeusjiang.com/2021-04-16--xiang-xie-e-de-seo--di-tou--wo-bu-shu-liao-static-site--ban-ben.html

**Comment by czqme** [comment]
这个插件很赞。我之前想使用 TiddlyWiki 来做笔记，但是因为 SEO 和 index.html 太大的问题就寻找其他方案，但是后来没有找到好的对 tiddler 生成 html 的方案就放弃了，目前只用来存储一些知识链接 ><

**Reply by ThaddeusJiang** [reply]
@lanlyhs  推荐使用 https://github.com/ThaddeusJiang/tiddlyjam

原作者不维护了，我 fork 了一下，添加了一些使用的 features。
